packages/nuxeo-platform-explorer-package/src/main/resources/package.xml

```diff
@@ -9,7 +9,7 @@
   <home-page>http://doc.nuxeo.com/x/VoVH</home-page>
   <target-platform>
     <name>@DISTRIBUTION_NAME@</name>
-    <version>[@NUXEO_PLATFORM_VERSION@, 2025.0)</version>
+    <version>[@NUXEO_PLATFORM_VERSION@, 2027.0)</version>
```

Comment:
NIT: I am sometimes thinking: what if we are asked to release a LTS 2026
`<version>[@NUXEO_PLATFORM_VERSION@, 2026.0)</version>`
won't hurt